cmd/fakemachine/main.go

```diff
@@ -15,6 +15,7 @@ type Options struct {
 	Memory      int      `short:"m" long:"memory" description:"Amount of memory for the fakemachine in megabytes"`
 	CPUs        int      `short:"c" long:"cpus" description:"Number of CPUs for the fakemachine"`
 	ScratchSize string   `short:"s" long:"scratchsize" description:"On-disk scratch space size (with a unit suffix, e.g. 4G); if unset, memory backed scratch space is used"`
+	ShowBoot    bool     `long:"show-boot" description:"Show boot/console messages from the fakemachine"`
```

Comment:
The console output was never shown on purpose. Though it's useful for debugging, e.g. on some setups we've seen kernel panics in jenkins (due to nested virtualisation instabilities).

I do wonder whether it makes sense to have a specific show boot option or whether it should be a more general debug option which turns it on.

Reply:
I copied the name from `autopkgtest-virt-qemu`.

Would you prefer the boot/console messages etc. to go to a log file? That might be less annoying (and people can use `--debug-log /dev/stderr` if that's what they want).

Reply:
I've left this as `--show-boot` until/unless we have a better name.